api/src/routes/tenants_sources.rs

```diff
@@ -95,6 +97,7 @@ pub async fn create_tenant_and_source(
         source_name,
         source_config,
     } = tenant_and_source;
+    root_span.record("project", &tenant_id);
```

Comment:
Is `tenant_id` always the project ref? Trying to understand how this works.

Reply:
Yes, a tenant in the `api` create is a project.